mohamad-dakdouk/week4/mon/D16-calwork-hm/calwork.rb

```diff
@@ -0,0 +1,165 @@
+puts "hi"
+
+# Calculator
+# Explanation
+
+#     You will be building a calculator which can perform multiple arithmetic operations. Your program should allow the user to choose which operation to perform, enter the values on which the operation is performed, and output the result.
+
+# Specification:
+
+#     A user should be given a menu of operations
+#     A user should be able to choose from the menu
+#     A user should be able to enter numbers to perform the operation on
+#     A user should be shown the result
+#     The menu should be displayed again, and process should continue until the user selects a quit option from the menu
+
+# Phase 1
+
+#     Calculator functionality
+#     Calculator should be able to do basic arithmetic (+,-, *, /)
+
+# Phase 2
+
+#     Advanced Calculator functionality
+#     Calculator should be able to do more complex arithmetic (exponents, square roots) - STFW
+
+# Bonus
+# Mortgage Calculator
+
+# Calculate the monthly required payment given the other variables as input (look up the necessary variables)
+# BMI Calculator
+
+# Calculate the body mass index (BMI) for an individual, given their height and weight
+# Trip Calculator
+
+# Calculate a trip time and cost given inputs for
+
+#     distance
+#     miles per gallon
+#     price per gallon
+#     speed in miles per hour
+
+
+
+
+def number1
+    print "enter number 1 :"
+    gets.to_i
+end
+
+def number2
+    print "enter number 2 :"
+    gets.to_i
+end
+
+def options
+    puts "Select for the following operations"
+    puts "1 is plus"
+    puts "2 is minus"
+    puts "3 is divide"
+    puts "4 is times"
+    puts "5 is exponent"
+    puts "6 is sqareroot"
+    print "Please insert the number here : "
+    gets.to_i
+end
+
+def plus
+    result = number1 + number2
+    print "#{result}"
```

Comment:
use `puts` instead as this will automatically create a new line
otherwise its `(result)Please insert the number here :` all squashed on one line